Source/Moya.swift

```diff
@@ -261,12 +305,12 @@ public func convertResponseToResult(response: NSHTTPURLResponse?, data: NSData?,
     }
 }
 
-private struct CancellableWrapper: Cancellable {
-    var innerCancellable: CancellableToken? = nil
+public struct CancellableWrapper: Cancellable {
+    public var innerCancellable: CancellableToken? = nil
```

Comment:
I was thinking, if we make this `internal`, then we can use `@testable import Moya` in our unit tests and avoid making this `public`. Thoughts?